### DISCUSSION TITLE
安装失败求解答

### DISCUSSION THREAD

**Discussion by ljmQAQ** [Richasy/Bili.Uwp#826] [问答]
应用安装失败，错误消息: 错误 0x8007000D: 发现名称为“”、mountPoint 为“C:”且媒体 ID 为“”的卷 1 损坏。 (0x8007000d)

出现这个怎么解决
![image](https://user-images.githubusercontent.com/101422521/157930672-4d30631f-e992-4aa2-8685-1ae3b4c77cf6.png)

**Comment by Richasy** [comment]
这个看上去像是系统磁盘的问题……应用这边解决不了
